2020_survivals/blog-posts/something-new.html

```diff
@@ -0,0 +1,306 @@
+                                        <div>
+                                            <i id="tag-icon" class="fas fa-tag link"></i>
+                                            <a class="link" href="../news.html">ALL</a>,<a class="link" href="#">CLIENTS</a>
+                                        </div>
+                                        <div>
+                                            <a class="link" href="#"><i class="fas fa-comment-alt"></i></a><span>  0</span>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="reply-section">
+                            <div class="reply-section-title">
+                                <h6 class="navi bold">Leave a reply</h6>
+                            </div>
+
+                            <form action="">
+                                <div class="form-group-lg">
+                                    <label for="message">Message</label><i class="fas fa-check-circle"></i><br>
+                                    <input class="form-control" type="text" id="message">
+                                </div>
+                                <div class="form-group-lg">
+                                    <label for="name">Name</label><i class="fas fa-check-circle"></i><br>
+                                    <input class="form-control" type="text" id="name">
+                                </div>
+                                <div class="form-group-lg">
+                                    <label for="email">Email</label><i class="fas fa-check-circle"></i><br>
+                                    <input class="form-control" type="email" id="email">
+                                </div>
+                                <div class="form-group-lg">
+                                    <label for="website">Website</label><br>
+                                    <input class="form-control" type="text" id="website">
+                                </div>
+                                <div class="form-check">
+                                    <input class="form-check-input" type="checkbox" value="" id="Check1">
+                                    <label class="form-check-label" for="Check"> Save my name, email, and website in this browser for the next time I comment.</label>
+                                </div><br>
+                                <label class="text-muted" for="alert"> Required fields are marked</label><i class="fas fa-check-circle"></i><br>
+                                <button type="submit" class="btn rounded-pill">SEND A COMMENT</button>
+                            </form>
+                        </div>
+                    </div>
+                    <div class="col col-12 col-md-12 col-lg-3">
+                        <div class="aside">
```

Comment:
You can also use the "<aside>" semantic HTML tag instead of this div with the aside class.